packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 43.5.0 - June 28, 2018
+
+### Fixed
+
+#### salesforcedx-vscode-core
+
+- Extensions failing to activate after an exception happens during setup ([PR #497](https://github.com/forcedotcom/salesforcedx-vscode/pull/497))
+
+#### salesforcedx-vscode-apex, salesforcedx-vscode-visualforce, salesforcedx-vscode-lwc
+
+- Specify a document selector for Apex, Visualforce, and LWC ([PR #502](https://github.com/forcedotcom/salesforcedx-vscode/pull/502))
```

Comment:
This change involves several extensions, do we want to break it per extension or should we not even have it as part of the changelog ?

Reply:
Maybe we can put it at the top level, under `salesforcedx-vscode`, even though it's technically for individual extensions? Also, how does this help users (beyond preventing confusing output in the console)?

Reply:
This is more of following vscode's suggested practices when the extension needs to interact with a file.

I'd be ok if everything is top level under `salesforcedx-vscode`.

Reply:
Let's put it at the top-level.

I don't think this directly affects the users for now. It does make our language server activation sequence safer since we only do it for files now and not other schemes.

Reply:
If it doesn't affect users, maybe it doesn't belong in the change log?

Reply:
Yeah, let's not include this one.